docs/lab/overview/quickstart.mdx

```diff
@@ -0,0 +1,285 @@
+     },
+     # Health Insurance is optional
+     health_insurance={
+          "front_image": {                    // Health Insurance (Optional)
+               "content": "base64-encoded-image-content",
+               "content_type": "image/png",
+          },
+          "back_image": {
+               "content": "base64-encoded-image-content",
+               "content_type": "image/png",
+          },
+     },
+)
+```
+
+</SnippetGroup>
+
+```json Response
+{
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6"),
+        "user_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "patient_details": {"dob": "2020-01-01", "gender": "male"},
+        "patient_address": {
+            "receiver_name": "John Doe",
+            "first_line": "123 Main St.",
+            "second_line": "Apt. 208",
+            "city": "San Francisco",
+            "state": "CA",
+            "zip": "91189",
+            "country": "United States",
+            "phone_number": "+1123456789",
+        },
+        "details": {
+            "type": "testkit",
+            "data": {
+                "id": "a655f0e4-6405-4a1d-80b7-66f06c2108a7",
+                "shipment": {
+                    "id": "d55210cc-3d9f-4115-8262-5013f700c7be",
+                    "outbound_tracking_number": "<outbound_tracking_number>",
+                    "outbound_tracking_url": "<outbound_tracking_url>",
+                    "inbound_tracking_number": "<inbound_tracking_number>",
+                    "inbound_tracking_url": "<inbound_tracking_url>",
+                    "outbound_courier": "usps",
+                    "inbound_courier": "usps",
+                    "notes": "<notes>",
+                    "created_at": "2020-01-01T00:00:00.000Z",
+                    "updated_at": "2020-01-01T00:00:00.000Z",
+                },
+                "created_at": "2020-01-01T00:00:00Z",
+                "updated_at": "2020-01-01T00:00:00Z",
+            },
+        },
+        "diagnostic_lab_test": {
+            "name": "Lipids Panel",
+            "description": "Cholesterol test",
+            "method": "testkit",
+        },
+        "sample_id": "123456789",
+        "notes": "This is a note",
+        "created_at": "2020-01-01T00:00:00Z",
+        "updated_at": "2020-01-01T00:00:00Z",
+        "status": "collecting_sample",
+        "events": [
+            {
+                "id": 1,
+                "created_at": "2022-01-01T00:00:00Z",
+                "status": "received.testkit.ordered",
+            }
+    ],
+}
+```
+
+As demonstrated in this Quickstart Guide, integrating with our API is incredibly straightforward, allowing you to access a wide range of lab tests with minimal effort. Placing an order for a lab test is as simple as making a request to the appropriate endpoint. With our user-friendly design and clear documentation, you can now harness the power of our clinical test API to enhance your application and deliver a seamless experience to your users.
```

Comment:
I would remove the adverbs again: `incredibly`. I would remove the "now" from "you can now harness", because it implies that before none of this was possible.